modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm

```diff
@@ -139,13 +161,40 @@
 	if(iscarbon(holder.my_atom))
 		data["injected_vol"] = data["injected_vol"] + passed_data["injected_vol"]
 		passed_data["injected_vol"] = 0
+	update_name()
 	..()
 
 /datum/reagent/synthtissue/on_new(passed_data)
 	if(!passed_data)
 		return ..()
 	if(passed_data["grown_volume"] > data["grown_volume"])
 		data["grown_volume"] = passed_data["grown_volume"]
+	update_name()
 	..()
 
+/datum/reagent/synthtissue/proc/update_name() //They are but babes on creation and have to grow unto godhood
+	switch(data["grown_volume"])
+		if(-INFINITY to 50)
+			name = "Induced Synthtissue Colony"
+		if(50 to 80)
+			name = "Oligopotent Synthtissue Colony"
+		if(80 to 135)
+			name = "Pluripotent Synthtissue Colony"
+		if(135 to 175)
+			name = "SuperSomatic Synthtissue Colony"
+		if(175 to INFINITY)
+			name = "Omnipotent Synthtissue Colony"
```

Comment:
Even less intuitive than the last one :laughing:
It's fine toh. Name em however you want.